website/docs/sdks/index.md

```diff
@@ -98,7 +98,7 @@ If you see an item marked with a ❌ that you would find useful, feel free to re
 | Can disable metrics | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
 | Client registration | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
 | Basic usage metrics (yes/no) | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| [Impression data](../advanced/impression-data.md) | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | N/A |
+| [Impression data](../advanced/impression-data.md) | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | ⭕ | ✅ | ⭕ | N/A |
```

Comment:
I believe this was added to the node SDK as well, by this PR: https://github.com/Unleash/unleash-client-node/pull/293

```suggestion
| [Impression data](../advanced/impression-data.md) | ⭕ | ✅  | ⭕ | ⭕ | ⭕ | ⭕ | ✅ | ⭕ | N/A |
```

Reply:
I think it would be out of scope, so maybe creating an issue to update it or another PR?

Reply:
@sighphyre From what I can tell, the PR only adds it as a field that they expect to get from the server. It doesn't look like they do anything with it, so I don't _think_ the node client supports impression data. I might be wrong, though.

Regardless, I agree with @RikudouSage that that should be a different PR in that case.

Reply:
Fair enough! Consider it ignored